### ISSUE TITLE
Stabilize pretty printed configurations

### ISSUE THREAD

**Issue by radumereuta** [runtimeverification/k#2211]
Is it possible to sort terms inside a pretty printed configuration from krun output?
Sometimes, when we make changes to seemingly unrelated parts, we get different [outputs](https://github.com/kframework/k/pull/2209/commits/ccff3618b1c548017b386975db7ff93dfad6f0f8).
We should be able to sort `Map`, `Set`, and cells with `multiplicity="*"`.

**Comment by radumereuta** [comment]
The sorting logic is in the llvm backend, kore-print binary.
A good place to start looking into this would be: https://github.com/kframework/llvm-backend/blob/master/tools/kprint/main.cpp#L250
```
sptr<KOREPattern> sorted = expanded->sortCollections(data);
```

**Comment by Baltoli** [comment]
The LLVM backend fix for this is in https://github.com/kframework/llvm-backend/pull/434. Once that change lands, the Jenkins PR to update the backend will need to be updated to incorporate the test reorderings.

**Comment by Baltoli** [comment]
Finally closed properly by #2254
